CODE_OF_CONDUCT.md

```diff
@@ -1,3 +1,168 @@
 The project follows the [GNOME Code of Conduct](https://wiki.gnome.org/Foundation/CodeOfConduct).
 
 If you believe that someone is violating the Code of Conduct, or have any other concerns, please contact us via [Discord](https://discord.gg/Sq85C42Xkt) or [Matrix](https://matrix.to/#/#warehouse:beeper.com).
+
+The GNOME Code of Conduct is also included at the end of this file.
+
+
+GNOME Code of Conduct
+
+Thank you for being a part of the GNOME project. We value your participation and want everyone to have an enjoyable and fulfilling experience. Accordingly, all participants are expected to follow this Code of Conduct, and to show respect, understanding, and consideration to one another. Thank you for helping make this a welcoming, friendly community for everyone.
```

Comment:
consider changing to "Thank you for being a part of the Warehouse project" and do that elsewhere as well?